pkg/requester/publicapi/wrapper_client.go

```diff
@@ -0,0 +1,155 @@
+package publicapi
+
+import (
+	"context"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/bacalhau-project/bacalhau/pkg/bidstrategy"
+	"github.com/bacalhau-project/bacalhau/pkg/job"
+	"github.com/bacalhau-project/bacalhau/pkg/model"
+)
+
+type RequesterAPIClientWrapper struct {
```

Comment:
The `RequesterAPIClientWrapper` wraps a `RequesterAPIClient`. `RequesterAPIClientWrapper` accepts `model` types as inputs, converts them to `v1beta2` types for its wrapped client, and converts `v1beta2` type returned from the wrapped client to `model` types.